test/migration_scripts_SUITE.erl

```diff
@@ -35,6 +35,7 @@ groups() ->
 
 init_per_suite(Config) ->
     file:delete("/tmp/script-debug"),
+    ok = stringprep:start(),
```

Comment:
Why do we need it here?

Reply:
jid module requires NIFs from this app, w/o it you will get `nif_not_loaded` error.

Reply:
Right, forgot that this PR uses `jid` in the suite.